docs/mom/files.md

```diff
@@ -0,0 +1,16 @@
+# [Kingdom Hearts Melody of Memory](index.md) File Overview
+
+## Extensions
+| Name | Purpose |
+|--|--|
```

Comment:
I suggest to use at least three hyphen, as our theme generator do not like less or equal than two.